code/src/components/DisplayAnswers.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Emoji } from './Emoji.js';
+
+export const DisplayAnswer = ({ answerName, answerDog, answerAmountOfDogs }) => {
+	const reloadPage = () => {
+		window.location.reload();
+	};
+
+	return (
+		<section className="questionContainer">
+			<h1>
+				<Emoji emoji="🐶 " />Your answers:
+			</h1>
+			<h3>What is your dogs name?</h3>
+			<h4>
+				<span className="underline">{answerName}</span>
+				<Emoji emoji="🐕‍🦺" />
+			</h4>
```

Comment:
This feels like it isn't a title of a section, more an answer that belongs to the title before?
```suggestion
			<p>
				<span className="underline">{answerName}</span>
				<Emoji emoji="🐕‍🦺" />
			</p>
```